simapp/.goreleaser.yml

```diff
@@ -0,0 +1,34 @@
+project_name: simd
+
+release:
+  disable: false
+  name_template: "{{.Tag}}"
+
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - main: ./simd
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_ENABLED=0
```

Comment:
do we need to set go tags as well like ledger?

Reply:
updated